Cargo.toml

```diff
@@ -29,6 +16,19 @@ members = [
 ]
 resolver = "2"
 
+[workspace.metadata]
+name = "fedimint"
+version = "0.1.0"
+authors = ["The Fedimint Developers"]
+edition = "2021"
+description = "Fedimint is a prototype Federated Chaumian E-Cash Mint implementation, natively compatible with Bitcoin & the Lightning Network. This project is under heavy development, DO NOT USE WITH REAL FUNDS."
+documentation = "https://github.com/fedimint/fedimint/tree/master/docs"
+readme = "README.md"
+homepage = "https://fedimint.org"
+repository = "https://github.com/fedimint/fedimint"
+license-file = "LICENSE"
+keywords = ["bitcoin", "lightning", "chaumian", "e-cash", "federated"]
```

Comment:
According to the cargo docs this meta data will just be ignored? I'm fine with leaving it in though, maybe some tools read it like this.